<launch>
    <arg name="speed_r" default="70" />
	<arg name="speed_l" default="70" />

    <node pkg="controls" type="controller" name="controller">
        <param name="speed_r" type="int" value="$(arg speed_r)" />
		<param name="speed_l" type="int" value="$(arg speed_l)" />
    </node>
    <node pkg="construction_check" type="tof_test" name="tof_test" />
    <node pkg="construction_check" type="ultrasonic_test" name="ultrasonic_test" />
    <node pkg="construction_check" type="drive_forward_backward_test" name="drive_forward_backward_test" />
    <node pkg="construction_check" type="drive_turn_test" name="drive_turn_test" />
    <node pkg="sensors" type="tof_sensor" name="tof_sensor" />
    <node pkg="sensors" type="ultrasonic_sensor" name="ultrasonic_sensor" />
</launch>
